sage_analysis/model.py

```diff
@@ -410,8 +410,8 @@ def read_sage_params(self, sage_file_path):
 
                         SAGE_dict[split[0]] = split[1]
 
-        except FileNotFoundError:
-            raise FileNotFoundError("Could not file SAGE ini file {0}".format(fname))
+        except FileNotFoundError as e:
+            raise e("Could not file SAGE ini file {0}".format(fname))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [e is not callable](https://app.codacy.com/manual/jacobseiler/sage-model/pullRequest?prid=4620074)